trojsten/results/templates/trojsten/results/view_results.html

```diff
@@ -7,26 +7,53 @@
 {% block page_header %}
 <h1>Výsledky<small>
 {% if single_round %}
-    {{ scoreboard.round.number }}. kola{% else %}po {{ scoreboard.round.number }}. kole
+    {{ round_score.round.number }}. kola{% else %}po {{ round_score.round.number }}. kole
```

Comment:
Táto premenná `round_score` sa používa len na to, aby si dostal dáta o kole.
Miesto nej pošli ako kontext rovno `round`.

Tiez ti to usetri to awkward posielanie `scoreboards[0][0]`